src/ui/public/vis/editor_size.js

```diff
@@ -0,0 +1,5 @@
+export const DefaultEditorSize = {
+  SMALL: 0,
```

Comment:
I think we could still use strings `'small'`, `'medium'`, etc. as values too, that would make the check in `default.html` a bit better readable.